quam_builder/architecture/quantum_dots/virtual_gates/README.md

```diff
@@ -0,0 +1,231 @@
+# VirtualGateSet: Abstracting Gate Control with Virtualisation Layers
+
+## 1. Introduction
+
+This document introduces the **VirtualGateSet** and **VirtualisationLayer** components. These extend the physical gate control capabilities provided by `GateSet` and `VoltageSequence` by adding one or more layers of virtual gates.
+
+Virtual gates simplify complex tuning procedures in experiments, especially for spin qubits, by allowing control over abstract parameters that map to multiple physical gate voltages.
+
+**`VirtualGateSet` builds upon `GateSet` and inherits all of its features**, including physical channel management, `VoltageTuningPoint` definitions, and all voltage control methods. This means you can use virtual gates while still having access to all the voltage control capabilities of the underlying `GateSet`.
```

Comment:
This sounds so amazing that I have to ask: does it mean that you can use the VoltageSequence with QUA variables for the levels and durations with virtual gates seamlessly?

Reply:
yes should be